tools/validators/instance_validator/tests/bad_fake_modified_ontology/FAKE_CARSON/entity_types/BAD_Carson.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This is an improperly written ontology file to test the validation in
```

Comment:
can you follow the same strucuture in the other unit tests

https://github.com/google/digitalbuildings/tree/master/tools/validators/ontology_validator/yamlformat/tests/fake_resources/BAD/entity_types

`fake_resource/BAD/...


Leave in the comments, what is wrong with the file: duplicate parent types for example

Reply:
Done